Makefile

```diff
@@ -0,0 +1,55 @@
+RANDOM_PORT := $(shell expr $$(( 8000 + (`id -u` % 1000) )))
+
+HUGO_VERSION ?= 0.54.0
+HUGO_URL := https://github.com/gohugoio/hugo/releases/download/v$(HUGO_VERSION)/hugo_$(HUGO_VERSION)_Linux-64bit.tar.gz
+HUGO_EXEC := vendor/hugo_$(HUGO_VERSION)
+
+CNAME := "dev-status.ocf.io"
+
```

Comment:
cstate is a theme for hugo; if we are git cloning and starting fresh, the theme git folder does not get populated. Below command does the job, would be nice to incorporate into Makefile, maybe `.PHONY: init`?
```
git submodule init
git submodule update 
```

Reply:
Yeah this is a fair point. I don't think we should put it in the Makefile but I was going to add it to the readme.